extensions/skins/conversation_v1/Conversation.js

```diff
@@ -410,10 +410,14 @@ oppia.directive('conversationSkin', [function() {
 
           $scope.explorationCompleted = oppiaPlayerService.isStateTerminal(
             stateName);
+
+          $scope.answerClassificationData =
```

Comment:
If I do this, I will still have to pass the state name to the interaction via parent inheritance (the interaction currently doesn't know the state name, and oppiaPlayerService needs the state name to get answerGroups), so I thought it was better to pass the answer groups and default outcomes directly. What do you think?